### ISSUE TITLE
install

### ISSUE THREAD

**Issue by kitten77** [iocage/iocage#122]
Installation

GitHub:

/usr/src is required to build from GitHub

pkg install python36
python3.6 -m ensurepip
++ pkg install git -- forget this one every time i have a new installasjon -hehe
git clone --recursive https://github.com/iocage/iocage
pip3.6 install Cython
cd iocage/py-libzfs && python3.6 setup.py build && python3.6 setup.py install
cd .. && pip3.6 install .
To install subsequent updates: git pull and pip3.6 install -U .

**Comment by skarekrow** [comment]
Ok are you saying I should add `git` to the install README? It's hard to parse the question :)

If so, I can certainly do so. I prefer `git-lite` to cut out dependencies. So I'll add that shortly.

**Comment by skarekrow** [comment]
And done! Thanks 👍

**Comment by kitten77** [comment]
Thank you :-)
